src/helm/benchmark/run_expander.py

```diff
@@ -306,7 +306,7 @@ class ModelRunExpander(ReplaceValueRunExpander):
         "full_functionality_text": get_model_names_with_tag(FULL_FUNCTIONALITY_TEXT_MODEL_TAG),
         "ai21/j1-jumbo": ["ai21/j1-jumbo"],
         "openai/curie": ["openai/curie"],
-        "openai/chat-gpt": ["openai/chat-gpt"],
+        "openai/chat-gpt": ["openai/chat-gpt", "openai/text-davinci-003"],  # Compare ChatGPT to text-davinci-003
```

Comment:
this seems like a confusing name. I think it makes plenty of sense to run both, but imo we should give a different name than just chatgpt if we are going to run both chatgpt and davinci

Reply:
Ah, that's a good point. I will update.